Resources/lang/fr/common.php

```diff
@@ -122,8 +122,8 @@
     'mtow'      => 'Max Take Off Weight',
     'mlw'       => 'Max Landing Weight',
     'mfuel'     => 'Fuel Capacity',
-    'mrange'    => 'Range',
-    'mseat'     => 'Seat Capacity',
+    'mrange'    => 'Rayon d\'action',
+    'mseat'     => 'Passagers',
```

Comment:
"Passagers" is "Passengers" but here we are talking about the capacity maybe "Capacité" or "Nombre de sièges" (number of seats)